server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/burstcoin/sources/poloniex/PoloniexRateSource.java

```diff
@@ -97,31 +90,27 @@ public BigDecimal getExchangeRateLast(String cryptoCurrency, String fiatCurrency
                 }
             }
         }
-
     }
 
     private BigDecimal getExchangeRateLastSync(String cryptoCurrency, String fiatCurrency) {
         if (!Currencies.NXT.equalsIgnoreCase(cryptoCurrency)) {
-            return null; //unsupported currency
+            return null; // unsupported currency
         }
-        OrderBookResponse orderBookResponse = api.returnOrderBook("returnOrderBook", "BTC_NXT", 10000);
+        OrderBookResponse orderBookResponse = api.returnOrderBook("returnOrderBook", "BTC_BURST", 10000); // Was NXT?
```

Comment:
Is this okay?

Reply:
Something that was missed, I think it needs to be changed.

Reply:
Should I leave as `BTC_BURST`?

Reply:
I do believe it should be BTC_BURST as it is the pair to call for said orderbook to get cost...